examples/tv-casting-app/APIs.md

```diff
@@ -810,21 +880,280 @@ let kDesiredEndpointVendorId: UInt16 = 65521;
+        DispatchQueue.main.async {
+            if err == nil {
+                self.connectionSuccess = true
+                self.connectionStatus = "Successfully connected to Casting Player!"
+            } else {
+                self.connectionSuccess = false
+                self.connectionStatus = "Connection to Casting Player failed with: \(String(describing: err))"
+            }
         }
-        else
+    }
+
+    let identificationDeclarationOptions: MCIdentificationDeclarationOptions
+    let targetAppInfo: MCTargetAppInfo
+    let connectionCallbacks: MCConnectionCallbacks
+    identificationDeclarationOptions = MCIdentificationDeclarationOptions()
+    targetAppInfo = MCTargetAppInfo(vendorId: kDesiredEndpointVendorId)
+    connectionCallbacks = MCConnectionCallbacks(
+        callbacks: connectionCompleteCallback,
+        commissionerDeclarationCallback: nil
+    )
+    identificationDeclarationOptions.addTargetAppInfo(targetAppInfo)
+
+    let err = selectedCastingPlayer?.verifyOrEstablishConnection(with: connectionCallbacks, identificationDeclarationOptions: identificationDeclarationOptions)
+    if err != nil {
+        self.Log.error("MCConnectionExampleViewModel connect(), MCCastingPlayer.verifyOrEstablishConnection() failed due to: \(err)")
+    }
+}
+```
+
+For a Casting Client to connect to a `CastingPlayer` using the optional
+`CastingPlayer` / Commissioner-Generated Passcode UDC feature, the Casting
+Client may specify optional parameters in the `VerifyOrEstablishConnection`
+function call, handle and then respond to the CastingPlayer's
+CommissionerDeclaration message as follows:
+
+1. In `VerifyOrEstablishConnection` the Casting Client should set the UDC
+   IdentificationDeclaration `CommissionerPasscode` field to true and provide a
+   `ConnectionCallbacks` callback to handle the CastingPlayer's
+   CommissionerDeclaration message.
+1. Upon receiving the CastingPlayer’s CommissionerDeclaration message with
+   PasscodeDialogDisplayed set to true, the Casting Client should prompt the
+   user to input the Passcode displayed on the `CastingPlayer` UX. If the user
+   declines to enter the Passcode, the Casting Client should call
+   `StopConnecting` to alert the `CastingPlayer` that the commissioning attempt
+   was canceled.
+1. The Casting Client should then update the commissioning session's PAKE
+   verifier with the user-entered Passcode using the APIs described in the
+   Initialize the Casting Client section above.
+1. Finally, the Casting Client should call `ContinueConnecting` to send a second
+   IdentificationDeclaration message to the `CastingPlayer` with
+   `CommissionerPasscodeReady` set to true.
+
+On Linux, the Casting Client can connect to a `CastingPlayer`, using the
+optional `CastingPlayer` / Commissioner-Generated Passcode UDC feature, by
+successfully calling `VerifyOrEstablishConnection`, updating the PAKE verifier
+and then calling `ContinueConnecting` on the `CastingPlayer`.
```

Comment:
From the client's perspective, they are just updating the passcode, so it may be more relevant to just use that verbiage.

```suggestion
successfully calling `VerifyOrEstablishConnection`, updating the set up passcode
and then calling `ContinueConnecting` on the `CastingPlayer`.
```